pandas/core/dtypes/cast.py

```diff
@@ -1269,14 +1270,20 @@ def convert_dtypes(
         Whether, if possible, conversion can be done to integer extension types.
     convert_boolean : bool, defaults True
         Whether object dtypes should be converted to ``BooleanDtypes()``.
+    convert_floating : bool, defaults True
+        Whether, if possible, conversion can be done to floating extension types.
+        If `convert_integer` is also True, preference will be give to integer
+        dtypes if the floats can be faithfully casted to integers.
```

Comment:
needs a version added, 
liking less and less the fact that we have many options here.

would change to an include=, exclude interface. (might involve deprecating the existing options). am -1 on doing this for 1.2 for this reason.

Reply:
> needs a version added,

This is the internal method, so doesn't need a versionadded. The public method already has a versionadded